content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,276 @@
+will serve as the single source of truth for our model management.
+
+To make use of our model registry, we will need some trained models first. To
+speed things up, we will start from a git
+[repository](https://github.com/iterative/example-get-started-model-management)
+with a model training pipeline already set up and ready to use.
+
+To see how you can track experiments and set up training pipelines with DVC and
+DVCLive, have a look at our getting started guide for
+[experiment management](/doc/start/experiments).
+
+## Adding models
+
+Let's now train a model and add it to the model registry. We will be using
+[DVCLive](/doc/dvclive) and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to save the
+model and add it to the model registry. Open the training script `src/train.py`
+in our example repository and you will see the following code under the
+`with Live(...)` context:
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in the Studio registry (other artifact types will not) and the rest of the
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+The training script runs as a part of a dvc pipeline
+[we prepared](/doc/start/experiments/experiment-pipelines) in the Experiment
+Management guide, so now we just call `dvc exp run` to run the experiment.
+
+We now commit the result to git (and push it to our git remote) and the new
+model will show up in the model registry in Studio. We will also push the
+resulting artifact to our DVC Remote by calling `dvc push`.
+
+<details id="push-click-to-see-how-artifacts-are-registered">
+
+#### 💡 Expand to see how adding models to the registry works under the hood
+
+When we call the `log_artifact()` method, DVC takes all the information we
+provided in the call and edits the `dvc.yaml` file which will now contain the
+following lines
+
+```yaml
+artifacts:
+  pool-segmentation:
+    path: models/model.pkl
+    type: model
+    desc:
+      This is a Computer Vision (CV) model that's segmenting out swimming pools
+      from satellite images.
+    labels:
+      - cv
+      - segmentation
+      - satellite-images
+```
+
+When you push the resulting file to your git remote it is parsed by Studio and
+it then shows all model artifacts from your `dvc.yaml` files in the model
+registry.
+
+</details>
+
+You should see something like the following picture in the Studio Model
+Registry.
```

Comment:
I think we need a bit more of an intro to Studio and clarify what role it plays.

Minor: I would move all the Studio stuff to the next section.